Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.13-alpine as builder
+FROM golang:1.14-alpine as builder
```

Comment:
strictly speaking, these version updates would normally not be part of a pull request.

Reply:
How do you mean? Only @thomseddon should make these changes?

Reply:
that's completely up to the maintainer, but in many projects I contributed to, a PR was really just about one specific feature or bugfix.

Reply:
reverted